src/DiscordChannel.php

```diff
@@ -31,16 +31,24 @@ public function __construct(Discord $discord)
      */
     public function send($notifiable, Notification $notification)
     {
-        if (! $channel = $notifiable->routeNotificationFor('discord', $notification)) {
+        if (!$channel = $notifiable->routeNotificationFor('discord', $notification)) {
```

Comment:
suspect this change will fail StyleCI check since the default in Laravel is with the space

Reply:
I have fixed it with another commit. I personally don't use StyleCI nor I have an integration in my IDE for that.